docs/UserGuide.adoc

```diff
@@ -17,14 +16,76 @@ endif::[]
 :contactsURL: https://contacts.google.com/
 :calendarURL: https://calendar.google.com/calendar/r/week
 
+= image:CodeducatorTitle.png[width=60px] Codeducator - User Guide
 By: `Team W09-B3`      Since: `Mar 2018`      Licence: `MIT`
 
-== Introduction
+
+== Welcome to Codeducator!
```

Comment:
This is great! Good job in adding content that can introduce our product to users more effectively and help them get familiarized with our user guide.